gcp/access_context_manager/service_perimeter_regular/locals.tf

```diff
@@ -1,11 +1,11 @@
 locals {
-  ingress_file      = try(fileexists(var.ingress_file_path), fileexists("./ingress_policies.yml")) ? file(var.ingress_file_path) : null
+  ingress_file      = fileexists(var.ingress_file_path) ? file(var.ingress_file_path) : null
   ingress_policies_read  = try(yamldecode(local.ingress_file), {})
-  ingress_policies = lookup(local.ingress_policies_read, "ingressPolicies", [])
+  ingress_policies = {for index, policy in lookup(local.ingress_policies_read, "ingressPolicies", []): index => policy}
```

Comment:
Changing from list of object to object with index as key, for use in for_each block